### PR TITLE
Fix gradle warnings

### LINKED ISSUE #116: Remove gradle KTS warnings

### PR DESCRIPTION
**Pull request by DeKaN**
Closes #116

### CODE REVIEW

**Comment by Kaaveh** [comment]
Thanks @DeKaN !
There is some warnings left that I'm mentioned here:
Screenshot 2023-10-05 at 7 53 31 AM

Please delete this empty file:
Screenshot 2023-10-05 at 7 51 40 AM

And remove these dead codes:
Screenshot 2023-10-05 at 7 50 26 AM

File: build.gradle.kts [hunk 1]
```diff
@@ -18,13 +18,13 @@ subprojects {
             if (project.findProperty("composeCompilerReports") == "true") {
                 freeCompilerArgs += listOf(
                     "-P",
-                    "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=${project.buildDir.absolutePath}/compose_compiler"
+                    "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=${project.layout.buildDirectory.asFile.get()}/compose_compiler"
```
Comment (Kaaveh) [inline build.gradle.kts:21]: Out of curiosity, why you changed this line? Could you please mention your reference?

**Review by Kaaveh — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (DeKaN) [reply build.gradle.kts:21]: img

**Comment by DeKaN** [comment]
@Kaaveh Fixed missed warnings

File: build.gradle.kts (hunk 1 above)
Comment (Kaaveh) [reply build.gradle.kts:21]: Interesting! In my Android studio I haven't this warning (Hedghog Beta 6). What's your Android studio Version?

**Review by Kaaveh — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (DeKaN) [reply build.gradle.kts:21]: It's deprecated by Gradle

**Comment by Kaaveh** [comment]
Thanks, @DeKaN for your Contribution! We hope to see more PR from you in the future! ❤️🙌🏻

**Pull request merged**